common/tags.go

```diff
@@ -0,0 +1,168 @@
+package common
+
+import (
+	"fmt"
+	"io"
+  "net/url"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/spf13/cobra"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	tagsFilePath string
+	tags map[string]string
+	argTags []string
+	tagsString string
+	ssmTags []*ssm.Tag
+	secretsManagerTags []*secretsmanager.Tag
+)
+
+func AddCommandWithTagging(parentCmd *cobra.Command, cmd *cobra.Command) {
```

Comment:
The wiring to the CLI should be confined to the `./cmd` package. I'd suggest we adjust some of the exported methods of this package to support this change. For example, if instead of `loadTagsFile` there was a `tagging.Config` type (or something similar) with proper `yaml` struct tags, the code in `./cmd` could use this type to unmarshal from a file when it is read from the command-line. The primary concern here is that the code here is highly coupled to a few other packages which makes testing it challenging. 

I'd suggest a refactor to make all the functionality in this package testable independently.